src/components/Balance/Balance.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const BALANCE = 1000000000000000000
+
+const Balance = (props) => {
+    let amount = props.amount / BALANCE
+    let style = {
+        width: "1.2em",
+        height: "1.2em",
+        marginLeft: "4px", 
+        bottom: "-2px",
+        position: "relative"
+    }
+    return (<div>
+        {(1234 < 0.01) ?
```

Comment:
This should be amount, right?

Reply:
yes, I just test with 1234

Reply:
just a tip: it's easier to test components, by passing props vs changing code in the component

Reply:
thank you